modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -123,7 +127,7 @@ def veteran_from_headers(with_gender: false)
         vet.gender = header('X-VA-Gender') || vet.gender_mpi if with_gender
         vet.edipi = vet.edipi_mpi
         vet.participant_id = vet.participant_id_mpi
-
+        vet.icn = !vet.mpi_icn.nil? ? vet.mpi_icn : nil
```

Comment:
Couldn't we do this with `&` as well and just let it return `nil` if it's not found?

Also, does this work with the new MPI stuff? I'd be curious to check, I think it was merged today.